certified-connectors/DocuSignDemo/script.csx

```diff
@@ -3405,9 +3405,9 @@ public class Script : ScriptBase
     if ("GetLoginAccounts".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase))
     {
       var uriBuilder = new UriBuilder(this.Context.Request.RequestUri);
-      var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
-      query["include_account_id_guid"] = "true";
-      uriBuilder.Query = query.ToString();
+      if(uriBuilder.Host.StartsWith("demo.")) uriBuilder.Host =  "account-d.docusign.com";
```

Comment:
You will have to add a condition for IL4 domain also (.mil)

Reply:
Do we support stage? I saw only demo connector and I thought we won't support is.

Reply:
Can you provide me the .mil domain url please. I can add it.

Reply:
There is different method for this already, I replaced it with that.